app/Services/Model/Imp/SummitOrderService.php

```diff
@@ -4280,9 +4282,38 @@ private function processInvitation(SummitOrder $order): SummitOrder
         Log::debug(sprintf("SummitOrderService::processInvitation trying mark invitation for email %s as accepted.", $order->getOwnerEmail()));
         $invitation->markAsAccepted();
 
+        $attendee = $this->attendee_repository->getBySummitAndEmail($summit, $order->getOwnerEmail());
+
+        if (!is_null($attendee)) {
+            CopyInvitationTagsToAttendees::dispatch($summit->getId(), $invitation->getId(), $attendee->getId());
+        }
+
         return $order;
     }
 
+    /**
+     * @inheritDoc
+     */
+    public function copyInvitationTagsToAttendee(int $summit_id, int $invitation_id, int $attendee_id):void {
+        $this->tx_service->transaction(function () use ($summit_id, $invitation_id, $attendee_id) {
+            $summit = $this->summit_repository->getById($summit_id);
+            if (!$summit instanceof Summit)
+                throw new EntityNotFoundException();
+
+            $attendee = $this->attendee_repository->getById($attendee_id);
```

Comment:
get the attendee from the summit, otherwise u could falsificate the relationship

Reply:
done